SQLite/Typed/Query.swift

```
@@ -955,7 +955,10 @@ extension Connection {
     }
 
     public func pluck(query: QueryType) -> Row? {
-        return try! prepare(query.limit(1, query.clauses.limit?.offset)).generate().next()
+        guard let rows = try? prepare(query.limit(1, query.clauses.limit?.offset)) else {
+            return nil
```

We should probably propagate the error up instead of returning `nil`.